docs/tutorial/part-one/index.md

```diff
@@ -14,7 +14,7 @@ Sit down, buckle up, and let's get going!
 
 ## Check your development environment
 
-Let's check first that you have everything setup to start creating with Gatsby.
+Let's check first that you have everything set up to start creating with Gatsby.
```

Comment:
`setup` is a noun. `set up` is the phrasal verb being used here.

Reply:
Amazing — never knew the difference 💯